rpc/src/v1/helpers/dispatch.rs

```diff
@@ -474,7 +474,7 @@ pub fn execute<D: Dispatcher + 'static>(
 					.map(ConfirmationResponse::SignTransaction)
 				).boxed()
 		},
-		ConfirmationPayload::Signature(address, mut data) => {
+		ConfirmationPayload::PostSignTransaction(address, mut data) => {
```

Comment:
The return type looks like it's just `H520` signature, no? And to sign arbitrary data you hash it first.
Not sure I follow, though...